hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/SqlProcessors.java

```diff
@@ -46,4 +50,16 @@ public static ProcessorSupplier rowProjector(
                         SimpleExpressionEvalContext.from(ctx)));
         return mapUsingServiceP(service, RowProjector::project);
     }
+
+    public static ProcessorSupplier rowProjector(
```

Comment:
Either:
- move `RowProjectorProcessorSupplier` to _this_ package, if it's generic enough
- or remove _this_ method as it references `IMap` specific supplier or nothing is/will be using it